packages/web-app/src/pages/PersonEditPage/PersonForm.jsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import {
+  FormControl as MuiFormControl,
+  TextField,
+  IconButton,
+  InputAdornment
+} from '@material-ui/core';
+
+import { Controller } from 'react-hook-form';
+
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import styled from 'styled-components';
+import { isEmpty, match } from 'ramda';
+
+import { emailRegexp } from '../../conf/Config';
+
+import StringInput from '../../components/common/Form/StringInput';
+
+const FormControl = styled(MuiFormControl)`
+  padding-bottom: ${({ theme }) => theme.spacing(4)}px;
+`;
+
+const PersonEditForm = ({ control, errors, watch }) => {
+  const [isPasswordVisible, setIsPasswordVisible] = React.useState(false);
+  const { formatMessage } = useIntl();
+
+  const toggleIsPasswordVisible = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+  const handleMouseDownPassword = event => {
+    event.preventDefault();
+  };
+
+  const validateName = value => {
+    if (value.length <= 0) {
+      return formatMessage({ id: 'You must have a name' });
+    }
+    return true;
+  };
+  const validateSurname = value => {
+    if (value.length <= 0) {
+      return formatMessage({ id: 'You must have a surname' });
+    }
+    return true;
+  };
+
+  const validatePwd = value => {
+    if (value.length > 0 && value.length < 8) {
+      return formatMessage({
+        id: 'Invalid email or password.'
+      });
+    }
+    return true;
+  };
```

Comment:
Pour des vérifications simples, react-hook-form intègre des "rules" de validation toutes prêts : 
List of validation rules supported:
  -  required
 -   min
  -  max
   - minLength
  -  maxLength
 -   pattern
-    validate

https://react-hook-form.com/get-started/#applyvalidation